tests/test_models.py

```diff
@@ -770,108 +819,7 @@ def _check_constraints(self):
         Entity and relation embeddings have to have at most unit L2 norm.
         """
         for emb in (self.model.entity_embeddings, self.model.relation_embeddings):
-            assert all_in_bounds(emb.weight.norm(p=2, dim=-1), high=1., a_tol=_EPSILON)
-
-    def test_score_hrt(self):
```

Comment:
why get rid of the custom tests?

Reply:
re-introduced. renamed to avoid masking the generic `test_score_hrt` test